### DISCUSSION TITLE
Run and package vela cli

### DISCUSSION THREAD

**Discussion by nitagarwal** [kubevela/kubevela#6087] [General]
We plan to use OAM and Kubevela as a runtime. We have few specific use cases. So, we plan to fork kubevela and modify it as per our requirements. 
We only plan to use the vela CLI and have currently no use case of VelaUX. We plan to use the ArgoCD for continuous deployment. 
Is there a documentation which outlines the step to create a installable release package from local code?

**Comment by Somefive** [comment]
You can try `make vela-cli` to build your local CLI to the `bin/` folder.

**Comment by wonderflow** [comment]
check the contribution guide: https://kubevela.io/docs/contributor/code-contribute#build
